hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/impl/ZookeeperFederationStateStore.java

```diff
@@ -325,47 +328,53 @@ public AddApplicationHomeSubClusterResponse addApplicationHomeSubCluster(
     FederationApplicationHomeSubClusterStoreInputValidator.validate(request);
     ApplicationHomeSubCluster app = request.getApplicationHomeSubCluster();
     ApplicationId appId = app.getApplicationId();
+    ApplicationSubmissionContext appSubmissionContext = app.getApplicationSubmissionContext();
+
+    LOG.info("appId = {}, homeSubClusterId = {}, appSubmissionContext = {}.", appId, app,
+        appSubmissionContext);
 
     // Try to write the subcluster
     SubClusterId homeSubCluster = app.getHomeSubCluster();
```

Comment:
Thank you very much for your help in reviewing the code! I want to record some log information about ApplicationSubmissionContext, so that we can check it when we need it, but the info level may be really unreasonable. I will change this part to debug level.